app/models/inbox_person.rb

```diff
@@ -4,6 +4,10 @@ class InboxPerson < ApplicationRecord
   # for some reason, the ActiveRecord plural for "Person" is "people"…
   self.table_name = 'inbox_persons'
 
+  # For historic reasons, we insert the registrant ID as the SQL table `id` column.
```

Comment:
I understand that this registrant ID is different from the actual registrant ID (that we use in WCIF). Is this ID named as registrantID anywhere? If not can we name it as personId or something else to avoid confusing with the actual registrantId?

Reply:
Please correct me if I'm wrong, but I believe that it is actually the registrantID

Reply:
Yes, confirmed that we are actually using the `registrantId` from WCIF. Scrambles-Matcher uses it to put that into the `id` field of the results JSON we are importing: https://github.com/thewca/scrambles-matcher/blob/7bf1013ef426e1ef5f770a0ffa155bf8ba2241c7/src/logic/import-export-wcif.js#L32-L33

Reply:
Oh my bad. I was under the assumption that `registrantId` is the primary key of `registrations` table. 🤦  Thanks for clarifying.